ocaml/xapi/xapi_fist.ml

```diff
@@ -122,3 +122,6 @@ let hang_psr psr_checkpoint =
       "psr_cleanup"
   )
   |> hang_while
+
+let exchange_certificates_among_all_members () =
```

Comment:
Would `exchange_certificates_in_pool` be clearer`? `all_members` makes me wonder if they are different from the pool.